### PR TITLE
chore: Small improvements while analysing ra2ce performance.

### PR DESCRIPTION
**Pull request by Carsopre**
Whilst analyzing code performance for ra2ce the following changes could be done:
- Removing of code smells.
- Streamlined (acceptance) tests.
- ...

### CODE REVIEW

File: ra2ce/graph/hazard/hazard_intersect/hazard_intersect_builder_for_tif.py [hunk 1]
```diff
@@ -71,9 +74,9 @@ def _from_networkx(self, hazard_overlay: Graph) -> Graph:
             *graph* (NetworkX Graph) : NetworkX graph with hazard values
         """
         # TODO apply multiprocessing?
-        from tqdm import (
-            tqdm,  # somehow this only works when importing here and not at the top of the file
-        )
+        # from tqdm import (
+        #     tqdm,  # somehow this only works when importing here and not at the top of the file
+        # )
```
Comment (ArdtK) [inline ra2ce/graph/hazard/hazard_intersect/hazard_intersect_builder_for_tif.py:79]: Should this code be removed?

File: ra2ce/graph/origins_destinations.py [hunk 2]
```diff
@@ -144,19 +144,25 @@ def closest_node(node: np.ndarray, nodes: np.ndarray) -> np.ndarray:
     return nodes[np.argmin(dist_2)]
 
 
-def get_od(o_id, d_id):
+def get_od(o_id: str, d_id: str) -> str:
```
Comment (ArdtK) [inline ra2ce/graph/origins_destinations.py:147]:
```suggestion
def get_od(o_id: str, d_id: str) -> str | np.nan:
```

File: ra2ce/graph/hazard/hazard_intersect/hazard_intersect_builder_for_tif.py (hunk 1 above)
Comment (Carsopre) [reply ra2ce/graph/hazard/hazard_intersect/hazard_intersect_builder_for_tif.py:79]: Yes.

File: ra2ce/graph/origins_destinations.py (hunk 2 above)
Comment (Carsopre) [reply ra2ce/graph/origins_destinations.py:147]: Unfortunately this is not accepted by Python, not sure why, this however does get accepted (`| type("np.nan")`). I will explore this in the future after the corresponding SonarCloud analysis.

**Review by ArdtK — APPROVED** [review]

**Pull request merged**